src/panels/AuditLogsFilterPanel/Fields/Fields.tsx

```diff
@@ -77,16 +81,19 @@ const Fields = (): JSX.Element => {
         as={AutocompleteInput}
         options={users}
       />
-      <Field
-        type="text"
-        // Have to manually set the id until this issue has been fixed:
-        // https://github.com/canonical/react-components/issues/957
-        id={Label.MODEL}
-        label={Label.MODEL}
-        name="model"
-        as={AutocompleteInput}
-        options={models}
-      />
+      {showModel ? (
+        <Field
+          type="text"
+          help='A model name in the format "controller-name/model-name"'
```

Comment:
Looked in other places and I think, for consistency reasons, it could be better to end the help message with a period.

Reply:
Done!